v2/internal/controllers/controller_resources_gen.go

```diff
@@ -227,27 +227,15 @@ func getKnownStorageTypes() []*registration.StorageType {
 	result = append(result, &registration.StorageType{
 		Obj: new(containerinstance_v20211001s.ContainerGroup),
 		Indexes: []registration.Index{
-			{
-				Key:  ".spec.containers.environmentVariables.secureValue",
-				Func: indexContainerinstanceContainerGroupContainersSecureValue,
-			},
-			{
-				Key:  ".spec.initContainers.environmentVariables.secureValue",
-				Func: indexContainerinstanceContainerGroupInitContainersSecureValue,
-			},
 			{
 				Key:  ".spec.imageRegistryCredentials.password",
 				Func: indexContainerinstanceContainerGroupPassword,
 			},
-			{
-				Key:  ".spec.diagnostics.logAnalytics.workspaceKey",
```

Comment:
This feels like something that should be a secret. Did it get deleted in a more recent API or something?

Reply:
Maybe separate out the structure.txt stuff from the submodule update, and we can do the submodule update in a separate PR?

Reply:
Fixed submodule reference and this change went away.